specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Pipeline.json

```diff
@@ -4312,6 +4316,10 @@
           "additionalProperties": {
             "type": "object"
           }
+        },
+        "storageIntegration":  {
+          "type": "object",
```

Comment:
The description says this is a string but the type here is `object`. Why is it not `string`?

Reply:
That's  for the syspase. In ADF and synapse share the same app model. And in ADF, we define it as the object, the reason is we support dynamic content which the actual value is resolved in service side.